core/app/c/(public)/[communitySlug]/public/forms/[formSlug]/fill/SubmitButtons.tsx

```diff
@@ -16,14 +16,11 @@ export const SubmitButtons = ({
 	// Use a default button if the user does not have buttons configured
 	if (buttons.length === 0) {
 		return (
-			<Button
-				id="submit-button-default"
-				type="submit"
-				disabled={isDisabled}
-				className={className}
-			>
-				Submit
-			</Button>
+			<div className={className}>
```

Comment:
the default button was unintentionally filling the whole width

Before:
![image](https://github.com/user-attachments/assets/58822602-a575-4551-9ad7-f06ceaa1f635)


After:
![image](https://github.com/user-attachments/assets/8edf2677-5514-4692-854d-36a1ee503ab3)